CHANGELOG

```diff
@@ -1,6 +1,9 @@
 qtile next_version:
-    !!! extention subpackage renamed to extension !!!
-    !!! extentions option renamed to extensions !!!
+    !!! `extention` subpackage renamed to `extension` !!!
+    !!! `widget_defaults` and `extentions` options merged into `extension_defaults` !!!
```

Comment:
Sure, I can think of `option_defaults`, `variable_defaults`, `arg_defaults`, `par_defaults`, simply `defaults`, and all the reversed `default_*` variants, e.g. `default_options` etc.